nilearn/__init__.py

```diff
@@ -44,8 +44,9 @@
 import sys
 import warnings
 
-from .version import __version__, _check_module_dependencies
+from .version import __version__
 
+# TODO: Check if still needed
```

Comment:
Can you deal with that ?

Reply:
@bthirion it seems to me that it can be removed but I'd rather deal with it in a different PR so that we could easily revert it if something breaks without reverting the packaging move.